Arelle/Arelle
# Adding NL BR KVK 2.02 validation

#### Reason for change

Added NL BR KVK 2.02 validation: FR-KVK-2.02: The attribute 'xml:lang' MUST contain the same value within an XBRL
instance document 

#### Steps to Test

Run validation against  tests/resources/validation/NL/fr_kvk/2.02/pass/valid.xml  and verify NO KVK 2.02 errors are returned. 

Run validation against  tests/resources/validation/NL/fr_kvk/2.02/fail/invalid.xml  and verify ONE KVK 2.02 error is returned.
**review**:
@Arelle/arelle
